app/build-types/experimental/manifest/_base.json

```diff
@@ -0,0 +1,5 @@
+{
+  "side_panel": {
```

Comment:
This would enable side panel for Firefox, which I think we don't want to do? We can probably delete this file, or leave it as an empty object if the build system expects it to exist.

Reply:
Removed it here https://github.com/MetaMask/metamask-extension/pull/36564/commits/f3ed24e60a2aa95ebec59b027eaf7df873888543